cert-gen

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env bash
+# note: Script uses -batch and -subj, instead of interactive prompts.
+# default environment variable values
+CERT_TEMP=./cert-tmp
+CA_CERT=$CERT_TEMP/ca.crt
+SVR_CERT=$CERT_TEMP/server.crt
+SVR_KEY=$CERT_TEMP/server.key
+SAN=DNS.1:localhost,IP.1:127.0.0.1
+
+set -e
+
+# Skip cert creation if the certs already exist
+if [ -d $CERT_TEMP ]
+then
+    echo "Certificate directory $CERT_TEMP exists. Skipping certificate creation."
```

Comment:
ic, that is a good secondary layer
we can do the proper Makefile dependency later

Reply:
ack